go/status/log_send.go

```diff
@@ -72,10 +72,23 @@ const redactedReplacer = "[REDACTED]"
 const serialPaperKeyWordThreshold = 5
 
 func redactPotentialPaperKeys(s string) string {
-	words := strings.Split(noncharacterRxx.ReplaceAllLiteralString(s, " "), " ")
+	doubleDelimited := noncharacterRxx.ReplaceAllFunc([]byte(s), func(x []byte) []byte {
+		return []byte{'~', '~', '~', x[0], '~', '~', '~'} // regexp is single char so we can take first elem
+	})
+	allWords := strings.Split(string(doubleDelimited), "~~~")
+	var checkWords []string
+	wordmap := make(map[int]int)
```

Comment:
Could you put a comment on this saying what it is? It took a few read throughs to figure out what it is.